indico_hub/server.py

```diff
@@ -158,6 +164,10 @@ def get_stats(
         'language': language,
         'debug': debug,
     }
+    # adding optional info and url
+    for field in kwagrs:
```

Comment:
kwagrs? that should be `kwargs` ;)

Reply:
is "kwargs" reserved by python?
I used kwagrs in the params so it sould work
changed it anyways !

Reply:
no you can use any name - the `**` in the argument list is what matters. but the current name is a typo so it should be fixed ;) (kwargs = keyword args)